addons/l10n_nl/models/account_chart_template.py

```diff
@@ -13,4 +13,12 @@ def _post_load_data(self, template_code, company, template_data):
                 company.account_journal_suspense_account_id.tag_ids += cash_tag
                 company.transfer_account_id.tag_ids += cash_tag
             if undist_profit_tag := self.env.ref('l10n_nl.account_tag_undist_profit', raise_if_not_found=False):
-                company.get_unaffected_earnings_account().tag_ids += undist_profit_tag
+                # All equity_unaffected accounts need to have the undist_profit_tag
```

Comment:
You are not setting the tag on "all equity_unaffected accounts", but you are creating an account with the tags instead